src/python/visclaw/geoplot.py

```diff
@@ -13,36 +13,45 @@
 # Colors
 black = [0.0,0.0,0.0]
 white = [1.0,1.0,1.0]
-red = [1.0,0.0,0.0]
 green = [0.0,1.0,0.0];
 dark_green = [0.1,0.4,0.0];
 light_green = [0.8,1.0,0.5];
-blue = [0.0,0.0,1.0];
 dark_blue = [0.2,0.2,0.7];
 light_blue = [0.5,0.5,1.0];
-blue_green = [0.0,1.0,1.0];
+blue = [0.0,0.0,1.0];
+blue_green = [0.0,1.0,1.0]
+red = [1.0,0.0,0.0]
 tan = [0.9,0.8,0.2];
 tan = [0.8,0.5,0.2];
 brown = [0.9,0.8,0.2];
 gray8 = [0.8,0.8,0.8];
 purple = [0.8,0.3,0.8];
 
+blue_a = [0.0,0.0,1.0,1.0];
+transparent = [0.0, 0.0, 0.0,0.0]
+red_a = [1.0,0.0,0.0,1.0]
+white_a = [1.0,1.0,1.0,1.0]
+
 # Colormaps
 TSUNAMI_MAX_AMPLITUDE = 0.6
+googleearth_colormap = colormaps.make_colormap({-TSUNAMI_MAX_AMPLITUDE:white_a,
```

Comment:
We really should not use an effectively sequential colormap for something that should be a divergent one.  See http://matplotlib.org/users/colormaps.html for more info.  I would favor perhaps using your newly defined `blue_a` as the bottom of this colormap.